code/__DEFINES/preferences_defines.dm

```diff
@@ -55,11 +55,11 @@
 // Yes I know this being an "enable to disable" is misleading, but it avoids having to tweak all existing pref entries
 #define PREFTOGGLE_2_REVERB_DISABLE					(1<<9) // 512
 #define PREFTOGGLE_2_FORCE_WHITE_RUNECHAT			(1<<10) // 1024
-#define PREFTOGGLE_2_SIMPLE_STAT_PANEL				(1<<11) // 2048
+// #define PREFTOGGLE_2_SIMPLE_STAT_PANEL			(1<<11) // 2048 Defunct as of 2024-02-14 with browser stat panels
```

Comment:
Not really in scope for this PR, but we have to figure out how to recycle these toggle prefs someday

Reply:
> Not really in scope for this PR, but we have to figure out how to recycle these toggle prefs someday
Datumize prefs :supercool:

Reply:
> Not really in scope for this PR, but we have to figure out how to recycle these toggle prefs someday

 It's called "create a script to toggle the bitflag off in the DB and then pester AA"